### PR TITLE
use the latest rocky 9 as default and new test training

### PR DESCRIPTION
**Pull request by mira-miracoli**

### CODE REVIEW

File: resources.yaml [hunk 1]
```diff
@@ -299,3 +298,10 @@ deployment:
     group: training-joca-epigenomics-24
     image: htcondor-secondary
     secondary_htcondor_cluster: true
+  training-rock:
+    count: 1
+    flavor: c1.c28m225d50
+    start: 2024-04-09
```
Comment (sanjaysrikakulam) [inline resources.yaml:304]:
```suggestion
    start: 2024-03-22
```

File: resources.yaml [hunk 2]
```diff
@@ -299,3 +298,10 @@ deployment:
...
+    end: 2024-03-22
```
Comment (sanjaysrikakulam) [inline resources.yaml:305]:
```suggestion
    end: 2024-04-09
```

**Review by sanjaysrikakulam — CHANGES_REQUESTED** [review]
start and end dates need to be swapped.

File: resources.yaml (hunk 1 above)
Comment (mira-miracoli) [reply resources.yaml:304]: no that was intentional, I thought there was no need to change the start date

Comment (sanjaysrikakulam) [reply resources.yaml]: But your start date is in April.

**Review by sanjaysrikakulam — APPROVED** [review]

**Pull request merged**